vehicle/mkv/software/bms/bms.yml

```diff
@@ -40,10 +40,17 @@ publish:
           name: V
           offset: 0
           scale: 0.0001 * 256
-      - name: pack_temperature
-        slice: 24 + 16
+      - name: min_temperature
+        slice: 24 + 8
         unit:
-          type: int16_t
+          type: uint8_t
+          name: degC
```

Comment:
Not actually degC. We need to figure out if we can linearize, or if we need to just report as voltages.